webapp/application.py

```diff
@@ -225,14 +229,12 @@ def faq():
 
 @application.route("/<string:token>")
 def application_index(token):
+    withdrawn = False
     application = _get_application_from_token(token)
-    if (
-        application["status"] != "active"
-        and application["rejection_reason"]["type"]["id"] == 2
-    ):
-        withdrawn = True
-    else:
-        withdrawn = False
+    print(application["rejection_reason"])
+    if application["status"] != "active" and application["rejection_reason"]:
+        if application["rejection_reason"]["type"]["id"] == 2:
```

Comment:
Is this the fixed line, were it used to through `KeyError` because of `application["rejection_reason"]` being equal to None in certain cases?

Reply:
Sorry, just read the PR's description I have the answer!

Reply:
Yes, if the candidate is hired the `application["rejection_reason"] == None`